src/course-outline/constants.js

```diff
@@ -1,9 +1,25 @@
+export const SECTION_BADGE_STATUTES = {
+  live: 'live',
+  publishedNotLive: 'published_not_live',
+  staffOnly: 'staff_only',
+  draft: 'draft',
+};
```

Comment:
```suggestion
export const SECTION_BADGE_STATUTES = /** @type {const} */ ({
  live: 'live',
  publishedNotLive: 'published_not_live',
  staffOnly: 'staff_only',
  draft: 'draft',
});
```
Do this and you'll get the correct types :)

![Screenshot 2023-12-08 at 3 10 58 PM](https://github.com/openedx/frontend-app-course-authoring/assets/945577/e5d3ee5b-0306-41ec-b9ab-53bebedbf54a)